Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx

```diff
@@ -330,6 +330,9 @@ void vtkSlicerPlaneRepresentation3D::UpdateFromMRML(vtkMRMLNode* caller, unsigne
 //----------------------------------------------------------------------
 void vtkSlicerPlaneRepresentation3D::UpdateInteractionPipeline()
 {
+  // Final visibility handled by superclass in vtkSlicerMarkupsWidgetRepresentation
```

Comment:
Add a note in the comment that explains why this call must be made first.

Reply:
Thanks! I've added a comment.